django/db/backends/base/creation.py

```diff
@@ -268,6 +272,13 @@ def destroy_test_db(self, old_database_name=None, verbosity=1, keepdb=False, num
             settings.DATABASES[self.connection.alias]["NAME"] = old_database_name
             self.connection.settings_dict["NAME"] = old_database_name
 
+    def _restore_initial_data_migrations(self):
+        # Load initial data migrations, if any.
+        if hasattr(connections[self.connection.alias], "_test_serialized_contents"):
+            connections[self.connection.alias].creation.deserialize_db_from_string(
+                connections[self.connection.alias]._test_serialized_contents
+            )
```

Comment:
Shouldn't we have a warning if somebody is using `--keepdb` option, but with `SERIALIZE` set to `False` in its database settings (I guess that will be a situation where we won't have any `_test_serialized_contents` ?).